internal/btp/cis/client.go

```diff
@@ -72,14 +72,6 @@ func (c *httpClient) post(url string, options requestOptions) (*http.Response, e
 	return c.genericRequest(http.MethodPost, url, options)
 }
 
-func (c *httpClient) put(url string, options requestOptions) (*http.Response, error) {
```

Comment:
Do we should remove these (probably unused) methods? IMO this client is like library and can have methods for potential use.

Reply:
I'd delete them, because I see two solutions:
- add linter ignore comment
    - when these function will get used, we may forget to remove this comment
- remove these functions
    - when somebody would want to use them, they would need to re-add the function (basically copy-paste of the post function)
Since this is a part of CIS package, where 9at least for now) we won't use any put or patch (or delete) methods, I'd remove these